app/assets/javascripts/reducers/stories.js

```diff
@@ -12,7 +15,47 @@ const storiesReducer = (state = initialState, action) => {
   switch (action.type) {
     case actionTypes.RECEIVE_STORIES:
       return action.data;
+    case actionTypes.CREATE_STORY:
+      if (isCreating(state)) {
```

Comment:
Should we have an `isCreating` being called inside an action called `CREATE_STORY`? There is something confusing here.